operator/scheduler/client.go

```diff
@@ -265,5 +250,24 @@ func (s *SchedulerClient) checkErrorRetryable(resource string, resourceName stri
 	} else {
 		return false
 	}
+}
 
+func retryFn(
+	fn func(context context.Context, conn *grpc.ClientConn, namespace string) error,
+	conn *grpc.ClientConn, namespace string, logger logr.Logger,
+) error {
+	logger.Info("RetryFn", "namespace", namespace)
+	logFailure := func(err error, delay time.Duration) {
+		logger.Error(err, "Scheduler not ready")
+	}
+	backOffExp := backoff.NewExponentialBackOff()
```

Comment:
I see the setting of `backOffExp.MaxElapsedTime = 0` is no longer present. The default is 15 minutes, just wanted to make sure the removal was intentional as the retries will now no longer go on forever (as in the comment to `startEventHandlers`.

Reply:
all calls via `retryFn` have an outer for loop that will make it indefinite also resetting the backoff state.

e.g.

```
		for {
			err := retryFn(s.SubscribeModelEvents, conn, namespace, s.logger)
			if err != nil {
				s.logger.Error(err, "Subscribe ended for model events", "namespace", namespace)
			} else {
				s.logger.Info("Subscribe ended for model events", "namespace", namespace)
			}
		}
```